libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapMatcherResult.kt

```diff
@@ -17,7 +17,9 @@ import com.mapbox.navigation.base.speed.model.SpeedLimit
  * @param isTeleport returns true if map matcher changed its opinion about most probable path on last update.
  * In practice it means we don't need to animate puck movement from previous to current location
  * and just do an immediate transition instead.
- * @param speedLimit current speed limit during free drive and active navigation session
+ * @param speedLimit current speed limit during free drive and active navigation session.
+ * In order to receive the speed limit make sure you add annotationsList with
+ * value "maxspeed" to the route request.
```

Comment:
```suggestion
 * In order to receive the speed limit make sure you add the with "maxspeed"
 * annotation to the route request.
```

Reply:
Might be better to suggest `DirectionsCriteria.ANNOTATION_MAXSPEED`.